pages/post/[postname].js

```diff
@@ -122,6 +122,21 @@ const StyledBlogPost = styled.div`
     img {
       max-width: 680px;
     }
+
+    ul,
+    ol {
+      margin-left: 3rem;
+    }
+
+    ul {
+      list-style: initial;
```

Comment:
Strange that you have to set this to initial...I haven't looked yet, but is something overriding it when it comes in? Otherwise, LGTM!

Reply:
I just checked the CMS, and it looks like double bullet points were added there. So we might not even need this?